taskweaver/planner/planner.py

```diff
@@ -188,6 +190,7 @@ def reply(
 
         def check_post_validity(post: Post):
             assert post.send_to is not None, "Post send_to field is None"
+            assert post.send_to != "Planner", "Post send_to field should not be Planner"
```

Comment:
Should add the error message to the self-correction message

Reply:
added